### DISCUSSION TITLE
Is this stack good for SaaS?

### DISCUSSION THREAD

**Discussion by mattveraldi** [epicweb-dev/epic-stack#888] [Q&A]
I'd like to use this stack for a SaaS that I am developing for a client because I'd rather use a solution like this to ship fast than to code everything from scratch and ship "whoknowswhen". 
I am not saying I have skill issues doing it, but I do have time issues 😂

I see that Epic Stack has pretty much everything I need, would you still recommend it? Is it still actively maintained?

**Comment by kentcdodds** [comment]
Hello @mattveraldi!

Yes, this is a great stack for a SaaS and plenty of people use it for that purpose.

And yes, it's actively maintained :)

Cheers!
